pkg/geo/geomfn/topology_operations.go

```
@@ -37,6 +37,26 @@ func Centroid(g geo.Geometry) (geo.Geometry, error) {
 	return geo.ParseGeometryFromEWKB(centroidEWKB)
 }
 
+// MinimumBoundingCircle returns minimum bounding circle of an EWKB
+func MinimumBoundingCircle(g geo.Geometry) (geo.Geometry, geo.Geometry, float64, error) {
+	polygonEWKB, centroidEWKB, radius, err := geos.MinimumBoundingCircle(g.EWKB())
+	if err != nil {
+		return geo.Geometry{}, geo.Geometry{}, 0, err
+	}
+
+	polygon, err := geo.ParseGeometryFromEWKB(polygonEWKB)
+	if err != nil {
+		return geo.Geometry{}, geo.Geometry{}, 0, err
+	}
+
+	centroid, _ := geo.ParseGeometryFromEWKB(centroidEWKB)
```

we should not ignore the error from here.